### PR TITLE
fixed search with 0

### PR DESCRIPTION
**Pull request by jeysmook**
If I want to search by 0 I get errors "Should not be blank or null values in kewywords field". I fixed it.

### CODE REVIEW

**Comment by chfabbro** [comment]
Hi @jeysmook. I appreciate the PR. We are doing some internal reorganization of the SDKs and who manages them, so it may take a little time before it can be merged.

**Comment by sivaschenko** [comment]
The issue description:

**Steps to reproduce:**
Use SDK to search for "0". (`SearchParameters::setWords("0")`)

**Actual Result:**
StockApiException 'Should not be blank or null values in kewywords field' is thrown

**Expected Result:**
"0" words search parameter is set successfully, as a result, it is possible to retrieve relevant search results for query "0"

**Additional Info:**

When `SearchParameters::setWords` is called with argument value `"0"` the condition `empty("0")` is resolved as "true" and the exception is thrown.

Also, as the `SearchParameters::setWords` function declaration has strict typed parameter `string` the function will not accept any parameter value that is not a string. (`TypeError` is thrown on an attempt to pass a value that is not string)

**Comment by jeysmook** [comment]
@chfabbro I added an update. Please see it.

File: src/Models/SearchParameters.php [hunk 1]
```diff
@@ -504,23 +504,18 @@ public function getWords() : ?string
...
      * Sets keywords in search Params for searching files.
      * @param string $words keywords that you want to search
      * @return SearchParameters object
-     * @throws StockApiException if words is null or blank
      */
     public function setWords(string $words) : SearchParameters
     {
-        if (empty($words)) {
-            throw StockApiException::WithMessage('Should not be blank or null values in kewywords field');
-        }
```
Comment (sivaschenko) [inline src/Models/SearchParameters.php]: Condition is removed as it is not needed :heavy_check_mark:

**Comment by chfabbro** [comment]
I can't accept it in this state. Somebody's IDE changed the formatting of every line to add additional space. Stock standards don't allow formatting changes without a separate ticket.

Please don't change the formatting when submitting PRs.

**Comment by jeysmook** [comment]
@chfabbro Hi! I modified updates. Please see it.

**Comment by chfabbro** [comment]
Looks good. Thank you for reverting the format changes!

**Pull request merged**